vocabularies/Org.OData.Core.V1.xml

```diff
@@ -509,12 +509,19 @@ Any simple identifier | Any type listed in `Validation.OpenPropertyTypeConstrain
         <Annotation Term="Core.LongDescription" String="The annotation value will usually be an expression, e.g. using properties of the binding parameter type for instance-dependent availability, or using properties of a singleton for global availability. The static value `null` means that availability cannot be determined upfront and is instead expressed as an operation advertisement." />
       </Term>
 
-      <TypeDefinition Name="LocalDateTime" UnderlyingType="Edm.String">
+      <Term Name="RequiresExplicitBinding" Type="Core.Tag" DefaultValue="true" AppliesTo="Action, Function">
+		<Annotation Term="Core.Description" String="This bound action or function is only available on model elements annotated with the ExplicitOperationBindings term." />
+      </Term>
+
+      <Term Name="ExplicitOperationBindings" Type="Collection(Edm.String)">
+        <Annotation Term="Core.Description" String="The qualified names of explicitly bound operations that are supported on the target model element. These bindings are in addition to any bindings to the type of the target model element not annotated with RequiresExplicitBinding." />

```

Comment:
What are "bindings ... annotated with `RequiresExplicitBinding`"? Should that be "actions/functions annotated with ..."?

Reply:
Good catch -- thanks!